apps/emqx_machine/src/emqx_machine.erl

```diff
@@ -61,17 +61,6 @@ set_backtrace_depth() ->
 is_ready() ->
     emqx_machine_terminator:is_running().
 
--if(?OTP_RELEASE > 22).
```

Comment:
So we won't warn users anymore even if it using an unsupported OTP version?

Reply:
it won't even compile on 22 now.